src/app/helptext/jails/jails-edit.ts

```diff
@@ -523,6 +532,12 @@ rtsold_tooltip: T('Use <a href="https://www.freebsd.org/cgi/man.cgi?query=rtsold
 target="_blank">rtsold(8)</a> as part of IPv6 autoconfiguration. \
 Send ICMPv6 Router Solicitation messages to interfaces to discover new routers.'),
 
+ip_hostname_placeholder: T('ip_hostname'),
+ip_hostname_tooltip: T(''),
+
+assign_localhost_placeholder: T('assign_localhost'),
+assign_localhost_tooltip: T(''),
```

Comment:
Set to add network interface <i>lo0</i> to the jail and assign it the first available localhost address, starting with <i>127.0.0.2</i>. <i>VNET</i> must be unset. Jails using <i>VNET</i> configure a localhost as part of their virtualized network stack.